src/app/component-library/policy-editor/model/policy-multi-expressions.ts

```diff
@@ -0,0 +1,28 @@
+import {UiPolicyExpressionType} from '@sovity.de/edc-client';
+
+export interface PolicyMultiExpressionConfig {
+  expressionType: UiPolicyExpressionType;
+  title: string;
+  description: string;
+}
+
+export const SUPPORTED_MULTI_EXPRESSIONS: PolicyMultiExpressionConfig[] = [
+  {
+    expressionType: 'AND',
+    title: 'AND',
+    description:
+      'Conjunction of several expressions. Evaluates to true iff all child expressions are true',
+  },
```

Comment:
"iff"

Reply:
fixed to non-abbreviated "if and only if"